### PR TITLE
JESI-3011

### PR DESCRIPTION
**Pull request by AndreTheHunter**
[JESI-3011](https://jesims.atlassian.net/browse/JESI-3011)

### CODE REVIEW

File: project.sh [hunk 1]
```diff
@@ -442,7 +447,7 @@ local-clean(){
 		mvn versions:display-dependency-updates &&
 		mvn versions:display-plugin-updates
 	fi
-	npm-cmd oudated
```
Comment (AndreTheHunter) [inline project.sh:445]: I did a Liz

File: project.sh [hunk 2]
```diff
@@ -572,14 +579,14 @@ local-clean(){
...
+			if [ -n "$watch" ];then
 				shadow-cljs compile karma
-				abort_on_error 'compiling test'
+				abort-on-error 'compiling test'
 				npx karma start --no-single-run --browsers=JesiChromiumHeadless &
 				shadow-cljs watch karma
 			else
 				shadow-cljs compile karma "${@:2}"
-				abort_on_error 'compiling test'
+				abort-on-error 'compiling test'
```
Comment (AndreTheHunter) [inline project.sh:603]: Fix

File: project.sh [hunk 3]
```diff
@@ -572,14 +579,14 @@ local-clean(){
 		-n|--node)
 			$cmd node "$@";;
 		*)
-			if [ $watch -eq 1 ];then
+			if [ -n "$watch" ];then
 				shadow-cljs compile karma
-				abort_on_error 'compiling test'
+				abort-on-error 'compiling test'
```
Comment (AndreTheHunter) [inline project.sh:598]: Fix

File: project.sh [hunk 4]
```diff
@@ -284,13 +284,18 @@ branch-name () {
 
 wait-for () {
 	local name=$1
-	local sleep=$2
+	local timeout=$2
 	local test_commands="${*:3}"
-	require-var name sleep test_commands
-	#TODO add timeout
```
Comment (AndreTheHunter) [inline project.sh:290]: TODone 😁

File: project.sh [hunk 5]
```diff
@@ -172,7 +172,21 @@ lein-dev () {
...
 
 lein-install () {
-	lein with-profile +install "$@"
+	lein with-profile +install,-dev "$@"
+}
+
+lein-jar(){
+	echo-message 'Building'
+	allow-snapshots
+	lein with-profile -dev jar "$@"
```
Comment (AndreTheHunter) [inline project.sh:181]: Excluding dev remove dev deps appearing in build

File: project.sh [hunk 6]
```diff
@@ -572,15 +593,16 @@ local-clean(){
...
 				shadow-cljs watch karma
 			else
 				shadow-cljs compile karma "${@:2}"
-				abort_on_error 'compiling test'
+				abort-on-error 'compiling test'
 				npx karma start --single-run --browsers=JesiChromiumHeadless
 			fi;;
 	esac
 }
+
```
Comment (AndreTheHunter) [inline project.sh]:
```suggestion
```

**Review by axrs — APPROVED** [review]

**Pull request merged**